definitions/variable/grids infrustructure/electricity_grid.yaml

```
@@ -0,0 +1,90 @@
+- Network|Electricity|Capacity:
+    description: Capacity of the line
+    unit: MW
+- Network|Electricity|Resistance
```

Missing semi-colon after the variable, also in other places.
```suggestion
- Network|Electricity|Resistance:
```